example/basic/docs/docs.go

```diff
@@ -1,18 +1,23 @@
 // GENERATED BY THE COMMAND ABOVE; DO NOT EDIT
-// This file was generated by swaggo/swag at
-// 2017-06-25 01:25:37.872454531 +0800 CST
+// This file was generated by swaggo/swag
 
 package docs
 
 import (
+	"bytes"
+	"encoding/json"
+	"strings"
+
+	"github.com/alecthomas/template"
```

Comment:
could I know why need to introduce `github.com/alecthomas/template` pkg?

Reply:
I created docs/docs.go by calling the command `swag init` (version 1.6.7)
I can minimize changes to the file if needed.

Reply:
I got it. It generated by command swag init. I didn't notice. 😆